third_party/jetifier/BUILD

```diff
@@ -3,11 +3,27 @@ java_import(
     jars = glob(["jetifier-standalone/lib/*.jar"]),
 )
 
+java_import(
+    name = "all_android_tools",
+    jars = ["@android_tools//:all_android_tools_deploy.jar"],
+)
+
+java_library(
+    name = "jetifier_worker_wrapper",
+    visibility = ["//visibility:public"],
+    srcs = glob(["**/*.java"]),
+    deps = [
+        "//third_party/jetifier:jetifier_jars",
+        ":all_android_tools",
+    ],
+)
+
 java_binary(
     name = "jetifier",
-    main_class = "com.android.tools.build.jetifier.standalone.Main",
+    main_class = "com.github.bazelbuild.rules_jvm_external.JetifierWorkerWrapper",
     visibility = ["//visibility:public"],
     runtime_deps = [
         ":jetifier_jars",
+        ":jetifier_worker_wrapper"
```

Comment:
Is it acceptable to copy WorkRequestHandler to this repo for the time being?

Reply:
That's one option, although also not ideal. There is an [implementation](https://github.com/bazelbuild/intellij/pull/3686/files) in the aswb plugin that's much simpler that we could pull into rules_jvm_external if we need to.

@SupSaiYaJin Are you using the `jetify_include_list` attribute on `maven_install` to limit the number of maven dependencies being Jetified to only those that need it? The screenshot in the description shows okio being Jetified which looks incorrect to me.

On a related note, AndroidX has been the default for 6+ years now. We should probably be figuring out how to remove [Jetifier](https://developer.android.com/tools/jetifier) entirely from `rules_jvm_external` instead of expanding support. People might even be unknowingly using it and experiencing slower builds as a result when it's in fact not needed. Alternatively we could switch the behavior to only run on the specific maven coords listed in `jetify_include_list` by default.

Reply:
Agree on that, we do use `jetify_include_list` to limit the libs. Besides even putting aside jetifier, this change could benefit all `java_binary` actions if they are stateless, for example
 https://github.com/bazelbuild/rules_jvm_external/blob/e6c1ff21e002bf97a7b1c07d63edd508a8dc9659/private/tools/java/com/github/bazelbuild/rules_jvm_external/jar/BUILD#L30

They could be disabled by default and enabled with `--strategy` if it fits consuming project needs.

I also found [BazelMultiplexWorker](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:bazel/src/com/android/tools/utils/BazelMultiplexWorker.java) with a nicer [API](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:bazel/src/com/android/tools/form/FormCompiler.java;l=61) `BazelMultiplexWorker.run(args, (compilerArgs, out) ->   );`